src/script/script.h

```diff
@@ -420,6 +420,7 @@ class CScript : public CScriptBase
 
     CScript& operator+=(const CScript& b)
     {
+        reserve(size()+b.size());
```

Comment:
Ultranit: spaces around `+`.

Reply:
Ooops. Fixed. Ran the clang-format snippet over it; also changed the declaration of "hex" in the test code.